common/arch/sdl/window.cpp

```diff
@@ -100,6 +100,10 @@ int window_close(window *wind)
 	return 1;
 }
 
+#if 0
```

Comment:
Why not just delete the function entirely?  It will still be in source control if we ever want the implementation back.